### PR TITLE
Make translations for new features

### PR DESCRIPTION
**Pull request by RisingInIris2017**
Thanks for your merging, and for your hard work making new features!
I have made these translations for all of these new things have been made.
Would you dear devs have them merged?
Thanks a lot! Praise the Angry Pixel!

### CODE REVIEW

File: src/main/resources/assets/thebetweenlands/lang/zh_CN.lang [hunk 1]
```diff
@@ -348,10 +349,21 @@ tile.thebetweenlands.nibbletwig_plank_stairs.name=奇趣枝楼梯
 tile.thebetweenlands.nibbletwig_plank_slab.name=奇趣枝台阶
 tile.thebetweenlands.nibbletwig_plank_fence.name=奇趣枝栅栏
 tile.thebetweenlands.nibbletwig_plank_fence_gate.name=奇趣枝栅栏门
-tile.thebetweenlands.amate_paper_pane_1.name=纸窗
+tile.thebetweenlands._paper_pane_1.name=纸窗
```
Comment (GoryMoon) [inline src/main/resources/assets/thebetweenlands/lang/zh_CN.lang]:
```suggestion
tile.thebetweenlands.amate_paper_pane_1.name=纸窗
```
This should be like this to work.

**Review by GoryMoon — APPROVED** [review]

File: src/main/resources/assets/thebetweenlands/lang/zh_CN.lang (hunk 1 above)
Comment (RisingInIris2017) [reply src/main/resources/assets/thebetweenlands/lang/zh_CN.lang]: Oops, thanks for your suggestion! 👍

**Comment by GoryMoon** [comment]
These are for a future update, the 3.3.13 was a hotfix update for a bug.
The bone Wayfinder is a magical thing.

**Comment by RisingInIris2017** [comment]
Thanks for your explanation!

**Pull request merged**